src/models/infobox-data.spec.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import {InfoboxData} from './infobox-data';
+import {isString} from '../validators';
+import {Caption} from '../types/caption';
+
+
+QUnit.module('Model');
+
+QUnit.test('InfoboxData constructor', function(assert) {
+     var id = new InfoboxData();
```

Comment:
When I saw `id` I thought "identifier". In other places we've used the var `data` to represent `InfoboxData` instances.